docroot/modules/custom/cs_generated_content/generated_content/node/civictheme_page_variations/02.automated_list.inc

```diff
@@ -1,35 +0,0 @@
-<?php
```

Comment:
This file was generating 50 nodes with the title "Listing" or later "Automated List" but not attaching any listings!